packages/augur-ui/src/modules/global-chat/components/global-chat.tsx

```diff
@@ -48,7 +50,9 @@ export const GlobalChat = () => {
             <span>Global Chat</span>
             <button onClick={() => setShow(!show)}>{Close}</button>
           </div>
-          <iframe src="./chat/index.html#/channel/augur" />
+          {theme === THEMES.TRADING && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
+          {theme === THEMES.BETTING && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
+          {theme === THEMES.SPORTS && <iframe src="./chat/index.html#/channel/augur" name={theme} />}
```

Comment:
Tried to reload a few different ways but I really need this iframe to dismount and remount in order for the theme to kick in.

If you have any other solutions, please let me know.

Reply:
useMemo, have it re-render on theme. so `const chat = useMemo(() => renderIframe, [theme])` then `{chat}` in render